projects/step-frontend/src/lib/modules/scheduler/components/scheduler-configuration/scheduler-configuration.component.html

```diff
@@ -2,10 +2,10 @@
   <mat-card-content>
     <step-form-field>
       <step-label>
-        Enable scheduler
+        Enable schedules
         <step-icon
           name="help-circle"
-          matTooltip="Used to enable or disable the scheduler globally accross all projects. Manual task execution remains possible."
+          matTooltip="Used to enable or disable the schedules globally accross all projects. Manual schedule execution remains possible."
```

Comment:
Keep "scheduler" and thus change to "Used to enable or disable the scheduler globally accross all projects. Manual schedule execution remains possible."